javadoc/pom.xml

```diff
@@ -395,6 +395,12 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-jspc-maven-plugin</artifactId>
       <scope>provided</scope>
+      <exclusions>
```

Comment:
This isn't going to help with #7677, as it's not the `javax.annotation` jar that is the problem, it is the `findbugs jsr305` jar that is the issue.